basedtyping/runtime_checks.py

```diff
@@ -0,0 +1,20 @@
+from types import UnionType
+from typing import Sequence, cast
+
+
+def issubtype(type1: type | UnionType, type2: type | UnionType) -> bool:
```

Comment:
`issubform`?

Reply:
resolving this because we already have [this thread](https://github.com/KotlinIsland/basedtyping/pull/7#pullrequestreview-842378852)